src/core/qgismobileapp.cpp

```diff
@@ -415,6 +416,7 @@ void QgisMobileapp::initDeclarative()
   qmlRegisterType<ValueMapModel>( "org.qfield", 1, 0, "ValueMapModel" );
   qmlRegisterType<RecentProjectListModel>( "org.qgis", 1, 0, "RecentProjectListModel" );
   qmlRegisterType<ReferencingFeatureListModel>( "org.qgis", 1, 0, "ReferencingFeatureListModel" );
+  qmlRegisterType<OrderedRelationModel>( "org.qgis", 1, 0, "OrderedRelationModel" );
```

Comment:
@m-kuhn , question: when is it proper to file types under org.qgis vs org.qfield (I was asking myself that question when implementing the digitizing logger stuff)

Reply:
QgsXxx classes under org.qgis
OtherClass under org.qfield
?

Reply:
@m-kuhn , makes sense.

Reply:
@m-kuhn  So I guess the three lines around me are done by mistake? If yes, I will change them in another PR.

Reply:
In case I'm responsible for `ReferencingFeatureListModel` then yes, I guess it's a mistake. I think it should have been `org.qfield`.